### ISSUE TITLE
Interface doesn't recognize approved cDAI

### ISSUE THREAD

**Issue by johnx25bd** [ProtektProtocol/protekt-frontend#5]
Testing out the app interfaces on Kovan at protekt.finance.

All worked well through the "Approve cDAI" transaction, but the interface did not update to let me transfer my cDAI to the contract when the Approve transaction validated.

Now when I log in with the address cDAI approved, I just get a spinner and can't actually transfer my cDAI to get pcDAI.

**Comment by stephenbyrne99** [comment]
Try kovan.protekt.finance ! Let me know if its all good and will close the issue :)

**Comment by johnx25bd** [comment]
The kovan.protekt.finance interface worked! I was able to both stake my cDAI to get pcDAI and stake WETH to shield mine.

One small thing - the visual feedback while the transaction is validating is good (slick notifications upper right), but the button doesn't automatically update from "Approve" to "Deposit" - I had to reload on both the Earn Safe Yield and Stake to Shield Mine pages to be able to Deposit. 

When the Approve transaction is verified, I would expect the interface to automatically update and pre-load the input field with the amount of cDAI or WETH I approved, so I just have one more click to Deposit and pop up Metamask.
